examples/gno.land/r/gov/dao/proxy/proxy.gno

```diff
@@ -0,0 +1,141 @@
+package proxy
+
+import "std"
+
+// dao is the actual govDAO implementation, having all the needed business logic
+var dao DAO
+
+// AllowedDAOs contains realms that can be used to update the actual govDAO implementation,
+// and validate Proposals.
+// This is like that to be able to rollback using a previous govDAO implementation in case
+// the latest implementation has a breaking bug. After a test period, a proposal can be
+// executed to remove all previous govDAOs implementations and leave the last one.
+var AllowedDAOs []string
+
+// proposals contains all the proposals in history.
+var proposals []*Proposal
```

Comment:
// TODO: change to use another structure instead of a slice to avoid storage problems.